Classes/Messages.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class Messages{
+    public static function setMsg($text, $type){
+        if($type = 'error'){
+            $_SESSION['errorMsg'] = $text;
+        } else {
+            $_SESSION['successMsg'] = $text;
+        }
```

Comment:
One equal sign sets the variable value and it does not check if it is equal to. For if to check if it is equal you need to use at least two equal sign `==`  or even better always use 3 equal sign `===`. Difference between two and three is that third equal sign checks also the type of both values, in your case both are I guess string.
Also try to avoid else, as it is usually not needed, you could just end of execution inside if with return.

But in your case you could use short if:
```suggestion
      $sessionKey = $type === 'error' ? 'errorMsg' : 'successMsg';
      $_SESSION[$sessionKey] = $text;
```